.pre-commit-config.yaml

```diff
@@ -6,9 +6,17 @@ repos:
       - id: check-yaml
       - id: check-added-large-files
         args: ["--maxkb=5000"]
+      - id: check-case-conflict
+      - id: check-merge-conflict
+      - id: check-symlinks
+      - id: debug-statements
+      - id: name-tests-test
+        args: ["--pytest-test-first"]
       - id: end-of-file-fixer
+      - id: mixed-line-ending
```

Comment:
What about using https://docs.astral.sh/ruff/settings/#format_line-ending instead?

Reply:
This will catch non-python files as well!

Reply:
Oh yes, let's keep this then!